Source/driver/reduction_nd.F90

```diff
@@ -63,6 +63,9 @@ subroutine reduce_min(x, y)
     real(rt), intent(in   ) :: y
     real(rt), intent(inout) :: x
 
+#ifdef _OPENMP
+    !$omp atomic update
```

Comment:
Normally we don't need this; this is a workaround for the fact that I removed the parallel reduction on the outer region. This routine is only still used by estdt_burning anyway, so when we switch to only the C++ interfaces, it will go away.